traces_browser.md

```diff
@@ -0,0 +1,14 @@
+### What is Traces Browser? 
+
+Explore the details of your application’s traces: 
+
+* Query for traces using the trace ID and span tags. 
+* Sort traces by the outliers, most errors, longest, and much more. 
```

Comment:
Sort traces by the outliers, most errors, longest, and much more. => Sort traces by outliers, most errors, longest, and more.